torch/quantization/ns/mappings.py

```diff
@@ -67,6 +66,7 @@
     torch.avg_pool1d,
     torch._C._nn.avg_pool2d,
     torch._C._nn.avg_pool3d,
+    torch.cat,
```

Comment:
looks like we also need to update the `TestFXNumericSuiteCoreAPIs.test_op_io_dtype_coverage` test for cat to point to this list of functions, instead of the old one